src/components/Setup/components/SetupWrapper.tsx

```
@@ -0,0 +1,222 @@
+ */
+const ImageSection = ({
+    layoutType,
+    image,
+    screenId,
+    imageClassName,
+}: Pick<SetupWrapperProps, 'layoutType' | 'image' | 'screenId' | 'imageClassName'>) => {
+    if (!image) return null
+
+    const isWelcomeOrSignup = layoutType === 'welcome' || layoutType === 'signup'
+    const containerClass = IMAGE_CONTAINER_CLASSES[layoutType]
+    const imageClass = !!imageClassName
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)